javascript/ql/src/Security/CWE-754/UnvalidatedDynamicMethodCall.qhelp

```diff
@@ -11,8 +11,8 @@ could choose a name that makes the application invoke an unexpected method, whic
 exception. If this exception is not handled, it could be used to mount a denial-of-service attack.
 </p>
 <p>
-For example, there might not be a method of the given name or the result of the lookup might not be
-a function, which would cause the method call to throw a <code>TypeError</code> at runtime.
+For example, there might not be a method of the given name, or the result of the lookup might not be
```

Comment:
line 35-36: add a comma for a better understanding:
Even if the object only contains methods**,** it is still a good idea to perform this check in case other properties are added to the object later on.